v2/example/active.bn.toml

```diff
@@ -0,0 +1,11 @@
+হ্যালোনাম = "Hello {{.Name}}"
```

Comment:
We don't have caps and small letters , what to do about it?
Hello = হ্যালো
Name = নাম